src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -209,6 +209,11 @@ public void submitWithoutConfirmationEmail() {
         clickSubmitButton();
     }
 
+    public boolean isConfirmationEmailBoxTicked() {
+        WebElement sendEmailChecbox = browser.driver.findElement(By.name(Const.ParamsNames.SEND_SUBMISSION_EMAIL));
```

Comment:
`sendEmailCheckbox`

Reply:
Hi @xpdavid 

I noticed this too but I realized the method `submitWithoutConfirmationEmail()` named this variable in this fashion too.

Not sure if it is intended but shall I fixed both variables to `sendEmailCheckbox` instead?

Reply:
I believe this is something not intended but a typo. You can fix that also.